### DISCUSSION TITLE
GitHub packages vs Artifactory/azure devops

### DISCUSSION THREAD

**Discussion by Dajma** [community/community#51102] [Packages]
How does GitHub packages compare to Artifactory and azure devops feeds? Artifactory and azure devops feeds have concepts of containers like folders for each kind of packages, so I can create a feed or a repository abc for maven packages and xyz for npm. Does GitHub packages have the same concept ? I don't see an option to create a package repository in GitHub. Do all of the different kinds of packages live in one folder named after the username/org name which can be accessed by all repositories owned by user/org?

**Comment by FlorentATo** [comment]
The main limitation I'm facing as we transition from Azure Devops to Github for package management is the lack of support of Universal Packages on Github. Cf. https://github.com/github/roadmap/issues/123
